packages/web-components/src/progress-bar/progress-bar.ts

```diff
@@ -0,0 +1,37 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTProgress } from '@microsoft/fast-foundation';
+import type { ProgressBarShape, ProgressBarThickness, ProgressBarValidationState } from './progress-bar.options.js';
+
+/**
+ * The base class used for constructing a fluent-progress-bar custom element
+ * @public
+ */
+export class ProgressBar extends FASTProgress {
+  /**
+   * The thickness of the progress bar
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: thickness
+   */
+  @attr
+  public thickness: ProgressBarThickness = 'medium';
```

Comment:
Do not set default values on the attributes, just document them in the jsDocs above.
See https://github.com/microsoft/fluentui/pull/26090#discussion_r1067181803 for more details.
I will add it to a "coding style" doc.